test/lorenz_delta.saved

```diff
@@ -0,0 +1,121 @@
+# Begin Test 0
+
+   braid_TestSpatialNorm:   Test 1 Passed
```

Comment:
Currently test 1 fails for me with this diff

    braid_TestBuf:   Test 1 Passed
 Finished braid_TestBuf: all tests passed successfully
    braid_TestInnerProd:   Test 1 Passed
-    braid_TestInitBasis:   Test 2 Passed
+    braid_TestInitBasis:   test 2 passed
    braid_TestBufBasis:   Test 1 Passed
    braid_TestBufBasis:   Test 2 Passed